route/table.go

```diff
@@ -153,7 +154,7 @@ func (t Table) addRoute(d *RouteDef) error {
 	switch {
 	// add new host
 	case t[host] == nil:
-		r := &Route{Host: host, Path: path}
+		r := &Route{Host: host, Path: path, Matcher: glob2.MustCompile(path)}
```

Comment:
What happens if this is an invalid pattern? Does it `panic`? You probably need to catch the error and return it.